src/OrbitGl/App.cpp

```diff
@@ -2723,41 +2720,6 @@ void OrbitApp::CaptureMetricProcessTimer(const orbit_client_protos::TimerInfo& t
   }
 }
 
-void OrbitApp::CaptureMetricProcessApiTimer(const orbit_client_protos::TimerInfo& timer) {
```

Comment:
Do we have the equivalent metrics somewhere else in the code base for the new code path? I don't see it.